packages/boinc/build.sh

```diff
@@ -1,22 +1,23 @@
 TERMUX_PKG_HOMEPAGE=https://boinc.berkeley.edu/
 TERMUX_PKG_DESCRIPTION="Open-source software for volunteer computing"
 TERMUX_PKG_LICENSE="GPL-3.0"
-TERMUX_PKG_VERSION=7.16.7
+TERMUX_PKG_VERSION=7.16.11
 TERMUX_PKG_SRCURL=https://github.com/BOINC/boinc/archive/client_release/${TERMUX_PKG_VERSION:0:4}/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=38130d532031e75701eee910da64b9eb837e5bfeff9979dbb200c37146be3fed
+TERMUX_PKG_SHA256=8a3f9453b16acfd8c636d18b4939c461c751aa1dd9e108cf60b90a3909bfa0a9
 TERMUX_PKG_DEPENDS="libandroid-shmem, libc++, libcurl, openssl, zlib"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_NO_STATICSPLIT=true
 
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --disable-server
---enable-client
```

Comment:
Sure about this?

Reply:
The default already has the value "--enable-client"
https://github.com/BOINC/boinc/blob/ebb0acebf97098336f89407258fbd89fc70f61bd/configure.ac#L68